core/src/components/list/list.ios.scss

```
@@ -19,12 +19,25 @@
   @include border-radius($list-inset-ios-border-radius);
 }
 
-.list-ios.list-inset ion-item:last-child {
+/**
+ * These selectors ensure the first item in the list
```

```suggestion
 * These selectors ensure the last item in the list
```
At least, I think this is what you meant? Ditto for the other instances of this comment.

---

Fixed in https://github.com/ionic-team/ionic-framework/pull/28439/commits/127e374bf65417bdadb178fff9378947257ca5e7